QUEST.tsv

```diff
@@ -87,13 +87,13 @@ QUEST_20150317_000086	The Goddess Vakarine left us shortly before Medis Diena. {
 QUEST_20150317_000087	{memo X}That's right. The strangest one of all was the Goddess Laima. {nl}No one's really sure when she disappeared.
 QUEST_20150317_000088	{memo X}Besides, isn't Laima the Goddess of Fate and Wisdom? {nl}She probably knew all about it.
 QUEST_20150317_000089	Resting Soldier
-QUEST_20150317_000090	I was an adventurer myself just like you, but then I got hit by an arrow…
+QUEST_20150317_000090	I used to be an adventurer like you. Then I took an arrow in the knee…
```

Comment:
I'm guessing that's the "liberal" translation... I wonder if they're allowed to make references to other games just like that.